hw/ip/spi_device/doc/_index.md

```diff
@@ -309,6 +361,252 @@ All reads and writes to/from the SRAM for RXF and TXF activity are managed by
 direct reads and writes through the TLUL bus interface, managed by the
 auto-generated register file control logic.
 
+## SPI Flash and Passthrough Modes
+
+### Command Information List
+
+The SW may configure the map from the received opcode to the command process module by programming *cmd_info* list.
+Current SPI_DEVICE provides 24 command information entries.
+Each entry represents a command.
+Details of the fields are explained in the {{<regref "CMD_INFO_0">}}
+
+First 11 commands are assigned to specific submodules.
+
```

Comment:
Can you document required encodings for the first 11 commands? As of now all reset values are 0, so it is important to have proper commands configured in the first 11 slots.

Reply:
OK. Since I don't see any reason to have other than the fixed value for Read Status, Read JEDEC ID, and Read SFDP, I may follow the same process as `EN4B` / `EX4B`, which means, SW just configure *valid* and *opcode*.

I will create an issue and follow up.